.gitignore

```diff
@@ -9,3 +9,5 @@ buildNumber.properties
 .mvn/timing.properties
 # https://github.com/takari/maven-wrapper#usage-without-binary-jar
 .mvn/wrapper/maven-wrapper.jar
+/src/test/resources/webdriver/chromedriver
+src/test/java/org/example/automation/amazon/BaseWebDriverTest.java
```

Comment:
Do not really understand why do you ignore that class

Reply:
Because the driver I use is different from other team member (I use Ubuntu ). So I need to change the class and replace the driver inside the chromedriver folder.
I  also put the details of it on the 2nd commit of this branch I created to make it clear. I hope it's understandable?